client/components/Home/index.jsx

```diff
@@ -1,76 +1,87 @@
-                            Working Mode
-                        </a>{' '}
-                        process.
+                        ARIA-AT is an interoperability testing project for
+                        assistive technologies like screen readers. The tests
+                        are based on design patterns from the W3C ARIA
+                        Authoring Practices Guide. By specifying the expected
+                        output given a specific user interaction, we can
+                        evaluate how consistently different ATs render user
+                        interface code.
                     </p>
-
-                    <h3>Get involved</h3>
-                    <ul>
-                        <li>
-                            <a href="https://www.w3.org/community/aria-at/">
-                                Join the community group
-                            </a>{' '}
-                            (see{' '}
-                            <a href="https://www.w3.org/community/about/faq/#how-do-i-join-a-group">
-                                how to join a W3C community group
-                            </a>
-                            )
-                        </li>
-                        <li>
-                            <a href="https://github.com/w3c/aria-at/wiki/How-to-contribute-tests">
-                                Write more tests
-                            </a>
-                        </li>
-                        <li>
-                            <a href="https://w3c.github.io/aria-at/">
-                                Review the assertions of existing test plans
-                            </a>
-                        </li>
-                        <li>
-                            <Link to="/results">Review test results</Link>
-                        </li>
-                        <li>
-                            <a href="https://github.com/w3c/aria-at/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22">
-                                Fix a good first issue
-                            </a>
-                        </li>
-                    </ul>
+                    <p>
+                        This app collects test data, hosts the latest test
+                        results reports, and provides an interface for
+                        manual testers to record results.
+                    </p>
+                    <div className="hero-buttons">
+                        <Button variant="primary">Sign Up to Run Tests</Button>
+                        <Button variant="secondary">Browse Test Reports</Button>
+                    </div>
+                </section>
+                <section className="get-involved">
+                    <div className="container">
+                        <h2>Get Involved</h2>
+                        <div className="resources">
+                            <article>
+                                <img src={iconJoinCommunity} alt=""/>
```

Comment:
just a reminder to add alt text to all of the images...

Reply:
I was thinking that since these images are abstractions and purely decorative, we could "hide" them for Screen Reders by having empty `alt` attributes

Reply:
TIL, thank you for explaining!